src/nvim/eval.c

```diff
@@ -9595,13 +9595,15 @@ static void f_getcwd(typval_T *argvars, typval_T *rettv, FunPtr fptr)
       if (from) {
         break;
       }
-    case kCdScopeTab:         // FALLTHROUGH
+      // fallthrough
```

Comment:
Well, it's comment that has special meaning, so making it look like a normal comment is misleading. But I don't feel strongly about it.